jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,88 @@
+describe('Index', function() {
```

Comment:
**Code Quality - Nitpick**

Add `use strict` since `src/inverted-index.js` has it too.